tests/cpp_unit_tests/test_tap_position_optimizer.cpp

```diff
@@ -990,47 +1006,6 @@ TEST_CASE("Test tap position optmizer I/O") {
                                                                           bad_regulators.end(), 50.0),
                         DuplicativelyRegulatedObject);
     }
-
-    SUBCASE("transformer tap position retrieval") {
```

Comment:
So the decision is to not unit test tap pos retrieval?

Reply:
it should be unit tested in `main_core` but here, the class' unit tests itself should take care of it. see https://github.com/PowerGridModel/power-grid-model/pull/607/files#diff-e3cb2a7b8db22693560826087f2c675630f9cd51a2337191aadc986dd50c4249R951-R958